sdk/python/kfp/compiler/compiler_test.py

```diff
@@ -3847,6 +3847,42 @@ def outer():
                 foo_platform_set_bar_feature(task, 12)
 
 
+class TestPipelineSemaphoreMutex(unittest.TestCase):
```

Comment:
split into two

Reply:
Done, split this test into two - one for semaphore and one for mutex.